Main.hs

```diff
@@ -79,7 +79,9 @@ boolExprBi _ _ _ = Error "Cannot perform binary boolean operation on non-strings
 -- Takes an expression; if it's a list, perform the given operation on it.
 -- Otherwise, throw an error.
 listExprUn :: ListOpUn -> Expr -> Expr
-listExprUn = undefined
+listExprUn (Ind i) (List v) = v !! i
+listExprUn (Map n) (List v) = undefined
```

Comment:
Mapping is not done yet, but I think this functionality is a soft requirement among the list operations required to implement.

Reply:
Yeah, but it shouldn't be required for the 50% mark. Mapping will require completed functions, which is not a part of the 50%.